netsuitesdk/internal/client.py

```diff
@@ -503,6 +506,16 @@ def upsert(self, record):
             return record_ref
         else:
             exc = self._request_error('upsert', detail=status['statusDetail'][0])
+
+            if record_type:
```

Comment:
It looks like this second param is only for error parsing. I think `upsertList` should be unaffected by this. We just won't get the benefits of whatever this does.

Reply:
I don't like the way this is implemented anyhow. You could pass "journal_entry" as the second param in an upsert to a Bill. I wonder what happens then.